web_external/User/InviteUserView.js

```diff
@@ -2,28 +2,40 @@ isic.views.InviteUserView = isic.View.extend({
     events: {
         'submit #isic-user-invite-form': function (event) {
             event.preventDefault();
-            // this.$('#isic-user-invite-submit').prop('disabled', true);
+            this.$('#isic-user-invite-submit').prop('disabled', true);
```

Comment:
I've just poked around a little bit, and most places seem to have it, which I didn't realize. I haven't exhaustively looked at every form, but I don't think a bug report is necessary any more.